tests/stomach_contents_tests.cpp

```diff
@@ -220,8 +220,8 @@ TEST_CASE( "hunger" )
         print_stomach_contents( dummy, print_tests );
         printf( "eat 2 cooked meat\n" );
     }
-    CHECK( hunger_time <= 270 );
-    CHECK( hunger_time >= 240 );
+    CHECK( hunger_time <= 320 );
```

Comment:
note that because of the way all of these are coded, they can only actually be multiples of 30.